tests/scripts/thread-cert/Cert_5_3_07_DuplicateAddress.py

```diff
@@ -101,26 +111,46 @@ def test(self):
         time.sleep(5)
         self.assertEqual(self.nodes[ROUTER2].get_state(), 'router')
 
-        self.nodes[ED1].start()
+        self.nodes[MED1].start()
         time.sleep(5)
-        self.assertEqual(self.nodes[ED1].get_state(), 'child')
+        self.assertEqual(self.nodes[MED1].get_state(), 'child')
 
-        self.nodes[ED2].start()
+        self.nodes[SED1].start()
         time.sleep(5)
-        self.assertEqual(self.nodes[ED2].get_state(), 'child')
+        self.assertEqual(self.nodes[SED1].get_state(), 'child')
 
-        self.nodes[ED3].start()
+        self.nodes[MED3].start()
         time.sleep(5)
-        self.assertEqual(self.nodes[ED3].get_state(), 'child')
+        self.assertEqual(self.nodes[MED3].get_state(), 'child')
+
+        # 2
+        leader_messages = self.sniffer.get_messages_sent_by(DUT_LEADER)
+        msg = leader_messages.next_mle_message(mle.CommandType.ADVERTISEMENT)
+        command.check_advertisement(msg)
```

Comment:
prefer to rename it to `check_mle_advertisement()` for better readability even not in concrete context.